doc/architecture/database_schemata.md

```diff
@@ -145,6 +191,70 @@ plant_detail }|--|| family : ""
+
+| **_Column name_** | **_Example_** | **_Description_**                                    |
+| :---------------- | :------------ | :--------------------------------------------------- |
+| **list_id**       | 1             | id of the ingredient list                            |
+| **plant_id**      | 1             | id of the plant used as an ingredient                |
+| **needed**        | 1             | the amount of this plant that is needed for the list |
+| **gathered**      | 0             | the amount of this plant that was already gathered   |
+
+## `Events`
+
+| **_Column name_** | **_Example_**        | **_Description_**                                                  |
+| :---------------- | :------------------- | :----------------------------------------------------------------- |
+| **id**            | 1                    | event id                                                           |
+| **map_id**        | 1                    | id of the map the event is taking place on                         |
+| **system_event**  | true                 | a flag representing whether this event is system or user generated |
+| **name**          | Harvest strawberries | name of the event                                                  |
+| **description**   | NULL                 | description of the event details                                   |
+| **event_date**    | 2023-04-15           | the date the event is taking place on                              |
+
+## `Favourites`
+
+Many-to-many table to store map-specific favourites.
+
+| **_Column name_** | **_Example_** | **_Description_** |
+| :---------------- | :------------ | :---------------- |
+| **map_id**        | 1             | id of the map     |
+| **user_id**       | 1             | id of the user    |
+| **plant_id**      | 1             | id of the plant   |
+
+## `CompanionGroups`
+
+| **_Column name_** | **_Example_**      | **_Description_**                                                         |
+| :---------------- | :----------------- | :------------------------------------------------------------------------ |
+| **id**            | 1                  | companion group id                                                        |
+| **name**          | My Companion Group | name of the companion group                                               |
+| **[user_id]**     | 1                  | id of the user that owns this companion group (are groups user specific?) |
+| **[map_id]**      | 1                  | id of the map that this companion group belongs to (map specific?)        |
+
+## `Companions`
+
+Many-to-many table to store relations between companion groups and plants.
+
+| **_Column name_** | **_Example_** | **_Description_**  |
+| :---------------- | :------------ | :----------------- |
+| **group_id**      | 1             | companion group id |
+| **plant_id**      | 1             | plant id           |
+
```

Comment:
What are these tables used for?

Reply:
I had the concept of plant companions stuck in my mind from earlier UI mockups. I went through the use cases again and it seems this is now covered by plant relationships. Since the table for that already exists, I will remove the companion tables.